ledger/store/schema.go

```diff
@@ -205,7 +205,7 @@ func accountsInit(tx *sql.Tx, initAccounts map[basics.Address]basics.AccountData
 
 		for addr, data := range initAccounts {
 			_, err = tx.Exec("INSERT INTO accountbase (address, data) VALUES (?, ?)",
-				addr[:], protocol.Encode(&data))
+				addr[:], protocol.Encode(&data)) //nolint:gosec // Encode does not hold on to reference
```

Comment:
How did you decide between this and some of the above changes that also don't hold references?

Reply:
I'm ok with it, just wondering if we should write down advice anywhere. I most often see the idiom
```
data := data
```
But that can imply a lot of copying sometimes.